src/Components/Component1/Footer.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function Footer(){
+    return (
+        {}//Nothing for now
```

Comment:
To render nothing you should use **null**. When you return empty object, browser fails with error